src/sdam/srv_polling.ts

```diff
@@ -67,7 +67,7 @@ export class SrvPoller extends TypedEventEmitter<SrvPollerEvents> {
     super();
 
     if (!options || !options.srvHost) {
-      throw new MongoDriverError('options for SrvPoller must exist and include srvHost');
+      throw new MongoInvalidArgumentError('Options for SrvPoller must exist and include srvHost');
```

Comment:
I think there is a different place where we validate the srvHost option from the user; I don't think they directly call SrvPoller so this would be a runtime error